src/components/application_manager/src/commands/hmi/on_exit_all_applications_notification.cc

```diff
@@ -75,6 +75,8 @@ void OnExitAllApplicationsNotification::Run() {
       break;
     }
     case hmi_apis::Common_ApplicationsCloseReason::SUSPEND: {
+      application_manager_.resume_controller().set_is_suspended(true);
+      application_manager_.resume_controller().OnSuspend();
```

Comment:
@LuxoftAKutsan yes, done in https://github.com/smartdevicelink/sdl_core/pull/1604/commits/1f59584e622b589eb2b4e17926b0cbb10463bcd9